OutOfSchool/OutOfSchool.BackgroundJobs/Extensions/Startup/ApplicationStatusChangingExtensions.cs

```diff
@@ -13,7 +15,10 @@ public static class ApplicationStatusChangingExtensions
     /// <param name="services">Services collection.</param>
     /// <param name="quartzConfig">Quartz configuration.</param>
     /// <exception cref="ArgumentNullException">Whenever the services collection is null.</exception>
-    public static void AddApplicationStatusChanging(this IServiceCollectionQuartzConfigurator quartz, IServiceCollection services, QuartzConfig quartzConfig)
+    public static void AddApplicationStatusChanging(
+        this IServiceCollectionQuartzConfigurator quartz,
+        IServiceCollection services,
```

Comment:
(Excessive parameter) Sorry, forgotten to mention it last time. `services` is not used, thus could be removed. Please check other extensions as well, as this seems to be copy/pasted everywhere